frontend/app/routes/$lang/_public/apply/$id/adult-child/review-child-information.tsx

```diff
@@ -190,48 +219,36 @@ export default function ReviewInformation() {
                       </p>
                     </DescriptionListItem>
                     <DescriptionListItem term={t('apply-adult-child:review-child-information.sin-title')}>
-                      {child.information.socialInsuranceNumber ? formatSin(child.information.socialInsuranceNumber) : ''}
+                      {child.sin ? formatSin(child.sin) : ''}
```

Comment:
Use `Logical && Operator`: https://react.dev/learn/conditional-rendering#logical-and-operator-

```suggestion
                      {child.sin && formatSin(child.sin)}
```